tracer/test/Datadog.Trace.TestHelpers/EnvironmentHelper.cs

```diff
@@ -496,7 +496,7 @@ public MockTracerAgent GetMockAgent(bool useStatsD = false)
             else
             {
                 // Default
-                var agentPort = TcpPortProvider.GetOpenPort();
+                var agentPort = fixedPort == -1 ? TcpPortProvider.GetOpenPort() : fixedPort;
```

Comment:
If you make `fixedPort` nullable (see my comment above), these lines can use the null-coalescing operator.
```suggestion
                var agentPort = fixedPort ?? TcpPortProvider.GetOpenPort();
```